executor/aggregate.go

```diff
@@ -1031,18 +1058,18 @@ func (e *HashAggExec) execute(ctx context.Context) (err error) {
 	}
 }
 
-func (e *HashAggExec) spillUnprocessedData(isFullChk bool) (err error) {
-	if isFullChk {
-		return e.listInDisk.Add(e.childResult)
-	}
+func (e *HashAggExec) spillUnprocessedData() (err error) {
 	for i := 0; i < e.childResult.NumRows(); i++ {
-		e.tmpChkForSpill.AppendRow(e.childResult.GetRow(i))
-		if e.tmpChkForSpill.IsFull() {
-			err = e.listInDisk.Add(e.tmpChkForSpill)
+		groupKey := e.groupKeyBuffer[e.childResult.GetRow(i).Idx()]
```

Comment:
Can we save the Row and use is in L1066? So we don't need to construct it again.